packages/mdadm/mdadm.spec

```diff
@@ -0,0 +1,44 @@
+%global debug_package %{nil}
+Name: %{_cross_os}mdadm
+Version: 4.3
+Release: 1%{?dist}
+Summary: mdadm is used for controlling Linux md devices (aka RAID arrays)
+License: GPL-2.0-only
+URL: https://cdn.kernel.org/pub/linux/utils/raid/mdadm/
+Source0: https://cdn.kernel.org/pub/linux/utils/raid/mdadm/mdadm-%{version}.tar.xz
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}systemd-devel
+
+%description
+%{summary}.
+
+%global set_env \
+%set_cross_build_flags \\\
+export CC=%{_cross_target}-gcc \\\
+CFLAGS="%{_cross_cflags} -DNO_COROSYNC -DNO_DLM" \
+LDFLAGS="%{_cross_ldflags}" \
+%{nil}
+
+%prep
+%autosetup -n mdadm-%{version} -p1
+
+%build
+%set_env
+make
+
+%install
+%set_env
+make install-bin DESTDIR=%{buildroot}%{_cross_rootdir}/usr
+make install-udev DESTDIR=%{buildroot}
+
+%files
+%license COPYING
+%{_cross_attribution_file}
+%{_cross_sbindir}/mdadm
+%{_cross_sbindir}/mdmon
+%{_cross_udevrulesdir}/01-md-raid-creating.rules
+%{_cross_udevrulesdir}/63-md-raid-arrays.rules
```

Comment:
These rules reference a few different systemd units:
* `mdmonitor.service`
* `mdmon@.service`
* `mdadm-grow-continue@.service`

It seems like all of those just call `mdadm` or `mdmon` so they should be fine to include. The first two aren't used for RAID-1 and it's doubtful that we'll need to grow a RAID-0 array, but better to have them unless we patch the udev rules to not need them.

Reply:
Added, required a few more changes for mdmonitor to actually work (specifying output goes to syslog and creating /var/run/mdadm).